### PR TITLE
Control verbosity

### PR DESCRIPTION
**Pull request by ftomassetti**
This PR introduces two flags to control the verbosity of the server. One controls the verbosity w.r.t. messages about database access, while the other one focus on logging requests

### CODE REVIEW

File: packages/core/src/controllers/LionWebBulkApi.ts [hunk 1]
```diff
@@ -6,6 +6,7 @@
 import { Request, Response } from "express"
 import { LionWebJsonChunk, LionWebValidator } from "@lionweb/validation"
 import { LIONWEB_BULKAPI_WORKER } from "./LionWebBulkApiWorker.js"
+import {requestsVerbosity} from "../server.js";
```
Comment (dslmeinte) [inline packages/core/src/controllers/LionWebBulkApi.ts:9]: Stylistic inconsistency: did you run `prettier`?

File: packages/core/src/controllers/LionWebBulkApi.ts [hunk 2]
```diff
@@ -20,7 +21,9 @@ class LionWebBulkApiImpl implements LionWebBulkApi {
      * @param res The list of all partition nodes, without children or annotations
      */
     async partitions(req: Request, res: Response): Promise<void> {
-        // const result = [];
+        if (requestsVerbosity) {
+            console.log(` * partitions request received, with body of ${req.headers["content-length"]} bytes`)
```
Comment (dslmeinte) [inline packages/core/src/controllers/LionWebBulkApi.ts:25]: Is it an idea to do all logging using an object with methods `requestLog`, `dbError`, etc. That object (and its methods) are aware of the value of `requestsVerbosity` etc. so these don't become separate global objects.

**Review by dslmeinte — CHANGES_REQUESTED** [review]

File: packages/core/src/controllers/LionWebBulkApi.ts (hunk 1 above)
Comment (ftomassetti) [reply packages/core/src/controllers/LionWebBulkApi.ts:9]: No. How can I do that? If you can tell me, I will run the command, add an entry in package.json and add a reference to it in the README

Comment (dslmeinte) [reply packages/core/src/controllers/LionWebBulkApi.ts:9]: Oh wait: `prettier` is not (yet) configured in this repo. That means you could fix this manually, or choose to forget about it ;)

Comment (joswarmer) [reply packages/core/src/controllers/LionWebBulkApi.ts:9]: There is a .prettierrc file in the repository, and the is a prettier dependency in the package.json. However, you need to run prettier by hand currently.

File: packages/core/src/controllers/LionWebBulkApi.ts (hunk 2 above)
Comment (joswarmer) [reply packages/core/src/controllers/LionWebBulkApi.ts:25]: I like the idea of a separate Logging object so we don't need the `if` around every log statement.

File: packages/core/src/controllers/LionWebBulkApi.ts (hunk 1 above)
Comment (ftomassetti) [reply packages/core/src/controllers/LionWebBulkApi.ts:9]: I ran prettier and that changed 69 files. So I would rather ignore stylistic issues in this file and open a separate PR where I add the task for prettier and I apply formatting to the whole codebase (see https://github.com/LionWeb-io/lionweb-repository/issues/28). Would this work for you?

File: packages/core/src/controllers/LionWebBulkApi.ts (hunk 2 above)
Comment (ftomassetti) [reply packages/core/src/controllers/LionWebBulkApi.ts:25]: I introduce the Logger object, as suggested

File: packages/core/src/controllers/LionWebBulkApi.ts (hunk 1 above)
Comment (joswarmer) [reply packages/core/src/controllers/LionWebBulkApi.ts]: Agreed!

**Review by dslmeinte — APPROVED** [review]

**Comment by ftomassetti** [comment]
Thank you @dslmeinte !

**Pull request merged**